src/ensembl/src/content/app/browser/browserConfig.ts

```diff
@@ -1,30 +1,20 @@
-import navigateUpIcon from 'static/img/browser/navigate-up.svg';
-import navigateUpOffIcon from 'static/img/browser/navigate-up-off.svg';
-
-import zoomOutIcon from 'static/img/browser/zoom-out.svg';
-import zoomOutOffIcon from 'static/img/browser/zoom-out-off.svg';
-
-import zoomInIcon from 'static/img/browser/zoom-in.svg';
-import zoomInOffIcon from 'static/img/browser/zoom-in-off.svg';
-
-import navigateDownIcon from 'static/img/browser/navigate-down.svg';
-import navigateDownOffIcon from 'static/img/browser/navigate-down-off.svg';
-
-import navigateLeftIcon from 'static/img/browser/navigate-left.svg';
-import navigateLeftOffIcon from 'static/img/browser/navigate-left-off.svg';
-
-import navigateRightIcon from 'static/img/browser/navigate-right.svg';
-import navigateRightOffIcon from 'static/img/browser/navigate-right-off.svg';
+import { ReactComponent as zoomInIcon } from 'static/img/browser/zoom-in.svg';
+import { ReactComponent as zoomOutIcon } from 'static/img/browser/zoom-out.svg';
+import { ReactComponent as navigateUpIcon } from 'static/img/browser/navigate-up.svg';
+import { ReactComponent as navigateDownIcon } from 'static/img/browser/navigate-down.svg';
+import { ReactComponent as navigateLeftIcon } from 'static/img/browser/navigate-left.svg';
+import { ReactComponent as navigateRightIcon } from 'static/img/browser/navigate-right.svg';
+import { ReactComponent as trackHeightBtn } from 'static/img/browser/icon_tracks_height.svg';
+import { ReactComponent as trackLockBtn } from 'static/img/browser/icon_tracks_lock_open.svg';
+import { ReactComponent as trackHighlightBtn } from 'static/img/browser/icon_tracks_highlight.svg';
+import { ReactComponent as trackMoveBtn } from 'static/img/browser/icon_tracks_move.svg';
```

Comment:
Are we okay to use `Btn` instead of `Button`?

@imransl @azangru @ens-ma7  any thoughts?

Reply:
these aren't even buttons (or btns 😃 ) — these are icons

Reply:
Changed to Icons.  :-) Thanks